ghost/core/core/server/services/link-redirection/LinkRedirectRepository.js

```diff
@@ -31,6 +31,22 @@ module.exports = class LinkRedirectRepository {
         linkRedirect.link_id = ObjectID.createFromHexString(model.id);
     }
 
+    async getAll(options) {
+        const collection = await this.#LinkRedirect.findAll(options);
+
+        const result = [];
+
+        for (const model of collection.models) {
+            result.push(new LinkRedirect({
+                id: model.id,
+                from: new URL(model.get('from'), this.#urlUtils.getSiteUrl()),
```

Comment:
This needs to correctly work with subdirectories, we need to trim the leading `/` from the from property